modules/op-running-unpriveleged-builds-in-a-container.adoc

```diff
@@ -0,0 +1,51 @@
+[id='op-running-unpriveleged-builds-in-a-container._{context}']
+= Running unprivileged builds in a container
+
+You can run builds as a non-root user inside a container. To run as a non-root user inside the container, your container must first add the `subuid` and `subgid` with the latest shadow-utils.
```

Comment:
Why would I want to though? I think we need to state the value of the running builds as a non-root user. Especially since the previous procedures and content posited that running containers a root user is a good, helpful thing.